doc/introduction/pennylane.rst

```diff
@@ -3,85 +3,45 @@
 
 .. _pl_intro:
 
-Introduction
-============
+What is PennyLane?
+==================
 
-PennyLane provides a bridge between classical and quantum computations, making it
-easy to build and optimize hybrid computations. Prominent examples are
-*variational quantum eigensolvers* and *quantum machine learning models*.
-
-Bridging the classical and quantum world
-----------------------------------------
-
-Classical computations, as well as optimization or training of models, are executed using
-one of the standard scientific computing or machine learning libraries. PennyLane provides an
-**interface** to these libraries, making them quantum-aware.
-
-PennyLane's standard interface is `NumPy <https://numpy.org/>`_,
-but interfaces to powerful machine learning libraries like `PyTorch <https://pytorch.org/>`_,
-`Tensorflow <https://www.tensorflow.org/>`_, and `JAX <https://github.com/google/jax>`__ are also available.
-
-The quantum computations are sent to a **device** for execution. A device can be real quantum
-hardware or a classical simulator. PennyLane comes with default simulator devices,
-but is well-integrated with external software and hardware to run quantum
-circuits---such as Xanadu's Strawberry Fields, Rigetti's Forest, IBM's Qiskit, or Google's Cirq.
-
-Communication between the classical computing library and quantum devices is coordinated in
-PennyLane via a suite of **plugins**:
-
-.. image:: ../_static/jigsaw.png
-    :align: center
-    :width: 90%
+.. figure:: ../_static/code.png
+    :align: left
+    :figwidth: 350px
+    :width: 300px
     :target: javascript:void(0);
 
 |
 
-The main job of PennyLane is to manage the evaluation of parametrized quantum circuits
-(so-called *variational circuits*) on quantum devices,
-and to make them accessible to the machine learning libraries.
-PennyLane also provides access to gradients of quantum circuits, which the machine
-learning library can use to perform backpropagation, including through quantum
-circuits---an essential process for optimization and machine learning.
+PennyLane is a cross-platform Python library for differentiable programming of
+quantum computers, enabling the execution and training of hybrid quantum-classical programs on various
+backends.
+
+PennyLane connects quantum computing with powerful machine learning frameworks
+like `NumPy <https://numpy.org/>`_'s `autograd <https://github.com/HIPS/autograd>`__,
+`JAX <https://github.com/google/jax>`__,
+`PyTorch <https://pytorch.org/>`_, and `TensorFlow <https://www.tensorflow.org/>`_,
+making them quantum-aware.
```

Comment:
We could also add scikit learn and scipy here?

Reply:
I'm not sure if this is fair - one can process data in scipy and put it into ANY framework after ;)

Reply:
True!